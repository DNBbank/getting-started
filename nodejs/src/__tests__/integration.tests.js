const {
  getTestCustomers,
  getCurrencyConversion,
  getCurrencyConversions,
} = require('..');
const loadCredentials = require('../credentials');

let hasCredentials = false;

try {
  const credentials = loadCredentials();
  console.log(credentials);
  hasCredentials = credentials.apiKey !== undefined;
} catch (error) {
  console.log(error);
}

function testRequiringCredentials(name, fn, timeout) {
  if (hasCredentials) {
    test(name, fn, timeout);
  } else {
    test.skip(name, () => Promise.reject(new Error('Missing API credentials')));
  }
}

testRequiringCredentials(
  'getTestCustomers should retrieve a list of test customers',
  async () => {
    expect(await getTestCustomers()).toMatchSnapshot();
  },
);

testRequiringCredentials(
  'getCurrencyConversions should currency info',
  async () => {
    const currencies = await getCurrencyConversions('NOK');

    expect(currencies).toMatchSnapshot();
  },
);

testRequiringCredentials(
  'getCurrencyConversion should currency info',
  async () => {
    const currencies = await getCurrencyConversion('EUR', 'NOK');

    expect(currencies).toMatchSnapshot();
  },
);
