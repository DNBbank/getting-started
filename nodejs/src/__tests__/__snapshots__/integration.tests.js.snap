// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getCards should retrieve list of cards 1`] = `
Array [
  Object {
    "accountNumber": "12050576967",
    "blockingInfo": Object {
      "blockAllowed": true,
      "performedBy": null,
      "timestamp": null,
      "unblockAllowed": true,
    },
    "cardHolderName": "Liv Ødegård",
    "cardId": "TQJQ95214468J85O",
    "cardStatus": "ACTIVE",
    "cardType": "DEBIT",
    "created": null,
    "creditBalance": null,
    "customerId": "29105573083",
    "debitBalance": null,
    "lastChanged": null,
    "maskedCardNumber": "************7196",
    "productName": "VISA CARD",
  },
  Object {
    "accountNumber": null,
    "blockingInfo": Object {
      "blockAllowed": false,
      "performedBy": null,
      "timestamp": null,
      "unblockAllowed": false,
    },
    "cardHolderName": "Liv Ødegård",
    "cardId": "IHMR56189598S24L",
    "cardStatus": "ACTIVE",
    "cardType": "CREDIT",
    "created": null,
    "creditBalance": null,
    "customerId": "29105573083",
    "debitBalance": null,
    "lastChanged": null,
    "maskedCardNumber": "************0855",
    "productName": "VISA CARD",
  },
  Object {
    "accountNumber": null,
    "blockingInfo": Object {
      "blockAllowed": false,
      "performedBy": null,
      "timestamp": null,
      "unblockAllowed": false,
    },
    "cardHolderName": "Liv Ødegård",
    "cardId": "RHFJ53721342S98L",
    "cardStatus": "ACTIVE",
    "cardType": "CREDIT",
    "created": null,
    "creditBalance": null,
    "customerId": "29105573083",
    "debitBalance": null,
    "lastChanged": null,
    "maskedCardNumber": "************6466",
    "productName": "VISA CARD",
  },
  Object {
    "accountNumber": null,
    "blockingInfo": Object {
      "blockAllowed": false,
      "performedBy": null,
      "timestamp": null,
      "unblockAllowed": false,
    },
    "cardHolderName": "Liv Ødegård",
    "cardId": "QLDW22575585E90E",
    "cardStatus": "ACTIVE",
    "cardType": "CREDIT",
    "created": null,
    "creditBalance": null,
    "customerId": "29105573083",
    "debitBalance": null,
    "lastChanged": null,
    "maskedCardNumber": "************2926",
    "productName": "VISA CARD",
  },
]
`;

exports[`getCurrencyConversion should currency info 1`] = `
Object {
  "baseCurrency": "EUR",
  "buyRateCash": 9.0478,
  "buyRateTransfer": 9.5047,
  "changeInRate": 1.17,
  "country": "EU",
  "midRate": 9.5407,
  "previousRateTransfer": 9.529,
  "quoteCurrency": "NOK",
  "sellRateCash": 10.04,
  "sellRateTransfer": 9.5767,
  "unit": 1,
  "updatedDate": "2018-11-06 09:00:00.0",
}
`;

exports[`getCurrencyConversions should currency info 1`] = `
Array [
  Object {
    "baseCurrency": "USD",
    "buyRateCash": 7.9999,
    "buyRateTransfer": 8.3266,
    "changeInRate": -1.25,
    "country": "US",
    "midRate": 8.3591,
    "previousRateTransfer": 8.3716,
    "quoteCurrency": "NOK",
    "sellRateCash": 8.7283,
    "sellRateTransfer": 8.3916,
    "unit": 1,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "EUR",
    "buyRateCash": 9.0478,
    "buyRateTransfer": 9.5047,
    "changeInRate": 1.17,
    "country": "EU",
    "midRate": 9.5407,
    "previousRateTransfer": 9.529,
    "quoteCurrency": "NOK",
    "sellRateCash": 10.04,
    "sellRateTransfer": 9.5767,
    "unit": 1,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "SEK",
    "buyRateCash": 87.79,
    "buyRateTransfer": 91.6,
    "changeInRate": 6.57,
    "country": "SE",
    "midRate": 92.3,
    "previousRateTransfer": 92.23,
    "quoteCurrency": "NOK",
    "sellRateCash": 96.82,
    "sellRateTransfer": 93,
    "unit": 100,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "DKK",
    "buyRateCash": 122.52,
    "buyRateTransfer": 127.19,
    "changeInRate": 17.81,
    "country": "DK",
    "midRate": 127.89,
    "previousRateTransfer": 127.72,
    "quoteCurrency": "NOK",
    "sellRateCash": 133.28,
    "sellRateTransfer": 128.59,
    "unit": 100,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "GBP",
    "buyRateCash": 10.3512,
    "buyRateTransfer": 10.8837,
    "changeInRate": 4.94,
    "country": "GB",
    "midRate": 10.9262,
    "previousRateTransfer": 10.8768,
    "quoteCurrency": "NOK",
    "sellRateCash": 11.6112,
    "sellRateTransfer": 10.9687,
    "unit": 1,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "CHF",
    "buyRateCash": 812.09,
    "buyRateTransfer": 830.8,
    "changeInRate": -91.31,
    "country": "CH",
    "midRate": 832.75,
    "previousRateTransfer": 833.66,
    "quoteCurrency": "NOK",
    "sellRateCash": 868.4,
    "sellRateTransfer": 834.7,
    "unit": 100,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "JPY",
    "buyRateCash": 7.0685,
    "buyRateTransfer": 7.354,
    "changeInRate": -1.95,
    "country": "JP",
    "midRate": 7.3765,
    "previousRateTransfer": 7.3961,
    "quoteCurrency": "NOK",
    "sellRateCash": 7.7345,
    "sellRateTransfer": 7.399,
    "unit": 100,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "ISK",
    "buyRateCash": 6.3354,
    "buyRateTransfer": 6.8184,
    "changeInRate": 2.9,
    "country": "IS",
    "midRate": 6.9184,
    "previousRateTransfer": 6.8894,
    "quoteCurrency": "NOK",
    "sellRateCash": 7.5014,
    "sellRateTransfer": 7.0184,
    "unit": 100,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "AED",
    "buyRateCash": 2.0887,
    "buyRateTransfer": 2.2677,
    "changeInRate": -0.35,
    "country": "AE",
    "midRate": 2.2757,
    "previousRateTransfer": 2.2792,
    "quoteCurrency": "NOK",
    "sellRateCash": 2.4627,
    "sellRateTransfer": 2.2837,
    "unit": 1,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "AUD",
    "buyRateCash": 5.5419,
    "buyRateTransfer": 6.0209,
    "changeInRate": 1.36,
    "country": "AU",
    "midRate": 6.0369,
    "previousRateTransfer": 6.0234,
    "quoteCurrency": "NOK",
    "sellRateCash": 6.5319,
    "sellRateTransfer": 6.0529,
    "unit": 1,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "BGN",
    "buyRateCash": 439.22,
    "buyRateTransfer": 0,
    "changeInRate": 62.07,
    "country": "BG",
    "midRate": 487.74,
    "previousRateTransfer": 487.12,
    "quoteCurrency": "NOK",
    "sellRateCash": 536.26,
    "sellRateTransfer": 494.74,
    "unit": 100,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "BHD",
    "buyRateCash": 0,
    "buyRateTransfer": 22.11,
    "changeInRate": -3.26,
    "country": "BH",
    "midRate": 22.18,
    "previousRateTransfer": 22.21,
    "quoteCurrency": "NOK",
    "sellRateCash": 0,
    "sellRateTransfer": 22.24,
    "unit": 1,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "BRL",
    "buyRateCash": 2.0229,
    "buyRateTransfer": 0,
    "changeInRate": -2.13,
    "country": "BR",
    "midRate": 2.2433,
    "previousRateTransfer": 2.2646,
    "quoteCurrency": "NOK",
    "sellRateCash": 2.5137,
    "sellRateTransfer": 2.2483,
    "unit": 1,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "BSD",
    "buyRateCash": 7.3562,
    "buyRateTransfer": 0,
    "changeInRate": -1.25,
    "country": "BS",
    "midRate": 8.3591,
    "previousRateTransfer": 8.3716,
    "quoteCurrency": "NOK",
    "sellRateCash": 9.362,
    "sellRateTransfer": 0,
    "unit": 1,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "BWP",
    "buyRateCash": 0,
    "buyRateTransfer": 0.7691,
    "changeInRate": 0.39,
    "country": "BW",
    "midRate": 0.7841,
    "previousRateTransfer": 0.7802,
    "quoteCurrency": "NOK",
    "sellRateCash": 0,
    "sellRateTransfer": 0.7991,
    "unit": 1,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "CAD",
    "buyRateCash": 6.1427,
    "buyRateTransfer": 6.3537,
    "changeInRate": -1.12,
    "country": "CA",
    "midRate": 6.3737,
    "previousRateTransfer": 6.3849,
    "quoteCurrency": "NOK",
    "sellRateCash": 6.7047,
    "sellRateTransfer": 6.3937,
    "unit": 1,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "CLP",
    "buyRateCash": 0.9444,
    "buyRateTransfer": 0,
    "changeInRate": 0.97,
    "country": "CL",
    "midRate": 1.2304,
    "previousRateTransfer": 1.2207,
    "quoteCurrency": "NOK",
    "sellRateCash": 1.5164,
    "sellRateTransfer": 0,
    "unit": 100,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "CNH",
    "buyRateCash": 0,
    "buyRateTransfer": 118.88,
    "changeInRate": -15.7,
    "country": "CN",
    "midRate": 120.78,
    "previousRateTransfer": 120.94,
    "quoteCurrency": "NOK",
    "sellRateCash": 0,
    "sellRateTransfer": 122.68,
    "unit": 100,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "CNY",
    "buyRateCash": 112.65,
    "buyRateTransfer": 118.83,
    "changeInRate": -13.69,
    "country": "CN",
    "midRate": 120.73,
    "previousRateTransfer": 120.86,
    "quoteCurrency": "NOK",
    "sellRateCash": 133.8,
    "sellRateTransfer": 122.63,
    "unit": 100,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "CZK",
    "buyRateCash": 34.11,
    "buyRateTransfer": 36.82,
    "changeInRate": 3.96,
    "country": "CZ",
    "midRate": 36.97,
    "previousRateTransfer": 36.93,
    "quoteCurrency": "NOK",
    "sellRateCash": 39.83,
    "sellRateTransfer": 37.12,
    "unit": 100,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "DOP",
    "buyRateCash": 0.1373,
    "buyRateTransfer": 0,
    "changeInRate": -0.02,
    "country": "DO",
    "midRate": 0.167,
    "previousRateTransfer": 0.1672,
    "quoteCurrency": "NOK",
    "sellRateCash": 0.1967,
    "sellRateTransfer": 0,
    "unit": 1,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "EGP",
    "buyRateCash": 0.2905,
    "buyRateTransfer": 0,
    "changeInRate": -0.1,
    "country": "EG",
    "midRate": 0.4665,
    "previousRateTransfer": 0.4674,
    "quoteCurrency": "NOK",
    "sellRateCash": 0.6425,
    "sellRateTransfer": 0,
    "unit": 1,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "ETB",
    "buyRateCash": 0,
    "buyRateTransfer": 0.2955,
    "changeInRate": -0.06,
    "country": "ET",
    "midRate": 0.2995,
    "previousRateTransfer": 0.3001,
    "quoteCurrency": "NOK",
    "sellRateCash": 0,
    "sellRateTransfer": 0.3035,
    "unit": 1,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "GMD",
    "buyRateCash": 12.22,
    "buyRateTransfer": 0,
    "changeInRate": -2.48,
    "country": "GM",
    "midRate": 16.59,
    "previousRateTransfer": 16.61,
    "quoteCurrency": "NOK",
    "sellRateCash": 20.95,
    "sellRateTransfer": 0,
    "unit": 100,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "HKD",
    "buyRateCash": 0.9679,
    "buyRateTransfer": 1.0619,
    "changeInRate": -0.15,
    "country": "HK",
    "midRate": 1.0669,
    "previousRateTransfer": 1.0684,
    "quoteCurrency": "NOK",
    "sellRateCash": 1.1659,
    "sellRateTransfer": 1.0719,
    "unit": 1,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "HRK",
    "buyRateCash": 118.39,
    "buyRateTransfer": 126.69,
    "changeInRate": 14.05,
    "country": "HR",
    "midRate": 128.29,
    "previousRateTransfer": 128.15,
    "quoteCurrency": "NOK",
    "sellRateCash": 138.2,
    "sellRateTransfer": 129.89,
    "unit": 100,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "HUF",
    "buyRateCash": 2.6033,
    "buyRateTransfer": 2.9413,
    "changeInRate": 0.88,
    "country": "HU",
    "midRate": 2.9663,
    "previousRateTransfer": 2.9575,
    "quoteCurrency": "NOK",
    "sellRateCash": 3.3393,
    "sellRateTransfer": 2.9913,
    "unit": 100,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "IDR",
    "buyRateCash": 0.0454,
    "buyRateTransfer": 0,
    "changeInRate": 0.06,
    "country": "ID",
    "midRate": 0.0564,
    "previousRateTransfer": 0.0559,
    "quoteCurrency": "NOK",
    "sellRateCash": 0.0674,
    "sellRateTransfer": 0.0574,
    "unit": 100,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "ILS",
    "buyRateCash": 2.0924,
    "buyRateTransfer": 0,
    "changeInRate": 0.4,
    "country": "IL",
    "midRate": 2.2684,
    "previousRateTransfer": 2.2644,
    "quoteCurrency": "NOK",
    "sellRateCash": 2.4744,
    "sellRateTransfer": 2.2984,
    "unit": 1,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "INR",
    "buyRateCash": 0.0947,
    "buyRateTransfer": 0.1125,
    "changeInRate": -0.01,
    "country": "IN",
    "midRate": 0.1145,
    "previousRateTransfer": 0.1146,
    "quoteCurrency": "NOK",
    "sellRateCash": 0.1343,
    "sellRateTransfer": 0.1165,
    "unit": 1,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "JOD",
    "buyRateCash": 10.68,
    "buyRateTransfer": 0,
    "changeInRate": -1.76,
    "country": "JO",
    "midRate": 11.78,
    "previousRateTransfer": 11.8,
    "quoteCurrency": "NOK",
    "sellRateCash": 12.88,
    "sellRateTransfer": 0,
    "unit": 1,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "KES",
    "buyRateCash": 6.5045,
    "buyRateTransfer": 0,
    "changeInRate": -2.04,
    "country": "KE",
    "midRate": 8.2315,
    "previousRateTransfer": 8.2519,
    "quoteCurrency": "NOK",
    "sellRateCash": 9.9585,
    "sellRateTransfer": 8.3715,
    "unit": 100,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "KRW",
    "buyRateCash": 0.6662,
    "buyRateTransfer": 0,
    "changeInRate": -0.16,
    "country": "KR",
    "midRate": 0.7432,
    "previousRateTransfer": 0.7447,
    "quoteCurrency": "NOK",
    "sellRateCash": 0.8202,
    "sellRateTransfer": 0.7532,
    "unit": 100,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "KWD",
    "buyRateCash": 0,
    "buyRateTransfer": 27.44,
    "changeInRate": -3.21,
    "country": "KW",
    "midRate": 27.52,
    "previousRateTransfer": 27.55,
    "quoteCurrency": "NOK",
    "sellRateCash": 0,
    "sellRateTransfer": 27.6,
    "unit": 1,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "LKR",
    "buyRateCash": 0,
    "buyRateTransfer": 0,
    "changeInRate": -0.44,
    "country": "LK",
    "midRate": 4.7944,
    "previousRateTransfer": 4.7989,
    "quoteCurrency": "NOK",
    "sellRateCash": 0,
    "sellRateTransfer": 4.8344,
    "unit": 100,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "MAD",
    "buyRateCash": 0,
    "buyRateTransfer": 0,
    "changeInRate": 4.43,
    "country": "MA",
    "midRate": 88,
    "previousRateTransfer": 87.96,
    "quoteCurrency": "NOK",
    "sellRateCash": 0,
    "sellRateTransfer": 89.5,
    "unit": 100,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "MKD",
    "buyRateCash": 0,
    "buyRateTransfer": 0,
    "changeInRate": -0.01,
    "country": "MK",
    "midRate": 0.1549,
    "previousRateTransfer": 0.155,
    "quoteCurrency": "NOK",
    "sellRateCash": 0,
    "sellRateTransfer": 0.3049,
    "unit": 1,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "MUR",
    "buyRateCash": 19.46,
    "buyRateTransfer": 0,
    "changeInRate": -0.82,
    "country": "MU",
    "midRate": 24.12,
    "previousRateTransfer": 24.13,
    "quoteCurrency": "NOK",
    "sellRateCash": 28.77,
    "sellRateTransfer": 0,
    "unit": 100,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "MVR",
    "buyRateCash": 0,
    "buyRateTransfer": 0,
    "changeInRate": -8.09,
    "country": "MV",
    "midRate": 54.07,
    "previousRateTransfer": 54.15,
    "quoteCurrency": "NOK",
    "sellRateCash": 0,
    "sellRateTransfer": 0,
    "unit": 100,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "MXN",
    "buyRateCash": 36.5,
    "buyRateTransfer": 0,
    "changeInRate": 36.4,
    "country": "MX",
    "midRate": 42,
    "previousRateTransfer": 41.64,
    "quoteCurrency": "NOK",
    "sellRateCash": 47.5,
    "sellRateTransfer": 42.75,
    "unit": 100,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "MYR",
    "buyRateCash": 1.6699,
    "buyRateTransfer": 0,
    "changeInRate": -1.16,
    "country": "MY",
    "midRate": 1.9988,
    "previousRateTransfer": 2.0105,
    "quoteCurrency": "NOK",
    "sellRateCash": 2.3277,
    "sellRateTransfer": 0,
    "unit": 1,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "NZD",
    "buyRateCash": 5.0197,
    "buyRateTransfer": 5.5537,
    "changeInRate": 0.09,
    "country": "NZ",
    "midRate": 5.5697,
    "previousRateTransfer": 5.5688,
    "quoteCurrency": "NOK",
    "sellRateCash": 6.1197,
    "sellRateTransfer": 5.5857,
    "unit": 1,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "OMR",
    "buyRateCash": 20.02,
    "buyRateTransfer": 0,
    "changeInRate": -3.25,
    "country": "OM",
    "midRate": 21.71,
    "previousRateTransfer": 21.74,
    "quoteCurrency": "NOK",
    "sellRateCash": 23.41,
    "sellRateTransfer": 21.76,
    "unit": 1,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "PHP",
    "buyRateCash": 12.92,
    "buyRateTransfer": 0,
    "changeInRate": 2.54,
    "country": "PH",
    "midRate": 15.76,
    "previousRateTransfer": 15.73,
    "quoteCurrency": "NOK",
    "sellRateCash": 18.59,
    "sellRateTransfer": 15.98,
    "unit": 100,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "PKR",
    "buyRateCash": 0,
    "buyRateTransfer": 0.0611,
    "changeInRate": -0.01,
    "country": "PK",
    "midRate": 0.0625,
    "previousRateTransfer": 0.0626,
    "quoteCurrency": "NOK",
    "sellRateCash": 0,
    "sellRateTransfer": 0.0639,
    "unit": 1,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "PLN",
    "buyRateCash": 194.99,
    "buyRateTransfer": 219.76,
    "changeInRate": 59.54,
    "country": "PL",
    "midRate": 221.66,
    "previousRateTransfer": 221.07,
    "quoteCurrency": "NOK",
    "sellRateCash": 248.34,
    "sellRateTransfer": 223.56,
    "unit": 100,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "QAR",
    "buyRateCash": 2.1083,
    "buyRateTransfer": 2.2873,
    "changeInRate": -0.4,
    "country": "QA",
    "midRate": 2.2953,
    "previousRateTransfer": 2.2993,
    "quoteCurrency": "NOK",
    "sellRateCash": 2.4823,
    "sellRateTransfer": 2.3033,
    "unit": 1,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "RON",
    "buyRateCash": 176.21,
    "buyRateTransfer": 0,
    "changeInRate": 20.73,
    "country": "RO",
    "midRate": 204.73,
    "previousRateTransfer": 204.53,
    "quoteCurrency": "NOK",
    "sellRateCash": 233.26,
    "sellRateTransfer": 206.73,
    "unit": 100,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "RSD",
    "buyRateCash": 6.5866,
    "buyRateTransfer": 0,
    "changeInRate": 0.58,
    "country": "RS",
    "midRate": 8.0601,
    "previousRateTransfer": 8.0543,
    "quoteCurrency": "NOK",
    "sellRateCash": 9.5336,
    "sellRateTransfer": 0,
    "unit": 100,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "RUB",
    "buyRateCash": 9.2717,
    "buyRateTransfer": 12.44,
    "changeInRate": 3.13,
    "country": "RU",
    "midRate": 12.67,
    "previousRateTransfer": 12.64,
    "quoteCurrency": "NOK",
    "sellRateCash": 16.07,
    "sellRateTransfer": 12.9,
    "unit": 100,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "SAR",
    "buyRateCash": 1.9865,
    "buyRateTransfer": 2.22,
    "changeInRate": -0.33,
    "country": "SA",
    "midRate": 2.2285,
    "previousRateTransfer": 2.2318,
    "quoteCurrency": "NOK",
    "sellRateCash": 2.4705,
    "sellRateTransfer": 2.237,
    "unit": 1,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "SGD",
    "buyRateCash": 5.543,
    "buyRateTransfer": 6.066,
    "changeInRate": -0.07,
    "country": "SG",
    "midRate": 6.082,
    "previousRateTransfer": 6.0827,
    "quoteCurrency": "NOK",
    "sellRateCash": 6.621,
    "sellRateTransfer": 6.098,
    "unit": 1,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "THB",
    "buyRateCash": 22.55,
    "buyRateTransfer": 25.05,
    "changeInRate": -2.64,
    "country": "TH",
    "midRate": 25.35,
    "previousRateTransfer": 25.38,
    "quoteCurrency": "NOK",
    "sellRateCash": 28.16,
    "sellRateTransfer": 25.65,
    "unit": 100,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "TND",
    "buyRateCash": 0,
    "buyRateTransfer": 0,
    "changeInRate": -3.86,
    "country": "TN",
    "midRate": 2.8929,
    "previousRateTransfer": 2.9315,
    "quoteCurrency": "NOK",
    "sellRateCash": 0,
    "sellRateTransfer": 2.9259,
    "unit": 1,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "TRY",
    "buyRateCash": 134.82,
    "buyRateTransfer": 0,
    "changeInRate": 366.81,
    "country": "TR",
    "midRate": 156.84,
    "previousRateTransfer": 153.17,
    "quoteCurrency": "NOK",
    "sellRateCash": 183.9,
    "sellRateTransfer": 160.64,
    "unit": 100,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "TWD",
    "buyRateCash": 0,
    "buyRateTransfer": 0,
    "changeInRate": -7.21,
    "country": "TW",
    "midRate": 27.18,
    "previousRateTransfer": 27.25,
    "quoteCurrency": "NOK",
    "sellRateCash": 0,
    "sellRateTransfer": 28.68,
    "unit": 100,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "TZS",
    "buyRateCash": 0,
    "buyRateTransfer": 0,
    "changeInRate": -0.05,
    "country": "TZ",
    "midRate": 0.3651,
    "previousRateTransfer": 0.3657,
    "quoteCurrency": "NOK",
    "sellRateCash": 0,
    "sellRateTransfer": 0.3741,
    "unit": 100,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "VND",
    "buyRateCash": 0.0248,
    "buyRateTransfer": 0,
    "changeInRate": -0.01,
    "country": "VN",
    "midRate": 0.0358,
    "previousRateTransfer": 0.0359,
    "quoteCurrency": "NOK",
    "sellRateCash": 0.0468,
    "sellRateTransfer": 0,
    "unit": 100,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
  Object {
    "baseCurrency": "ZAR",
    "buyRateCash": 0.4449,
    "buyRateTransfer": 0.5759,
    "changeInRate": 0.6,
    "country": "ZA",
    "midRate": 0.5879,
    "previousRateTransfer": 0.5819,
    "quoteCurrency": "NOK",
    "sellRateCash": 0.7309,
    "sellRateTransfer": 0.5999,
    "unit": 1,
    "updatedDate": "2018-11-06 09:00:00.0",
  },
]
`;
